config/importer.go

```diff
@@ -35,14 +38,16 @@ func (c *Config) String() string {
 // Load load the configuration & apply defaults where necessary
 func Load() (*Config, error) {
 	cfg := Config{
-		BindAddr:                 ":21000",
-		DatasetAPIAddr:           "http://localhost:22000",
-		DatasetAPIAuthToken:      "FD0108EA-825D-411C-9B1D-41EF7727F465",
-		DatabaseURL:              "bolt://localhost:7687",
-		PoolSize:                 20,
-		KafkaAddr:                []string{"localhost:9092"},
-		DimensionsExtractedTopic: "dimensions-extracted",
-		DimensionsInsertedTopic:  "dimensions-inserted",
+		BindAddr:               ":21000",
+		DatasetAPIAddr:         "http://localhost:22000",
+		DatasetAPIAuthToken:    "FD0108EA-825D-411C-9B1D-41EF7727F465",
+		DatabaseURL:            "bolt://localhost:7687",
+		PoolSize:               20,
+		KafkaAddr:              []string{"localhost:9092"},
+		IncomingInstancesTopic: "dimensions-extracted",
+		OutgoingInstancesTopic: "dimensions-inserted",
+		EventReporterTopic:     "event-reporter",
+		ShutdownTimeout:        time.Duration(5),
```

Comment:
After reading the function doc for time.duration is looks like this will be a duration of 5 nanoseconds? Consider changing this to `time.Second * 5`

Reply:
No. It returns the duration as nanoseconds. `time.Duration(5)` returns 5000000000 nanoseconds which is 5 seconds.